src/icons/style.css

```diff
@@ -17,6 +17,7 @@
   line-height: 1;
 
   /* Better Font Rendering =========== */
+  text-rendering: geometricPrecision;
```

Comment:
Could we scope this to `[class^="narmi-icon"]`? 

Chrome will render all text too light or too thin for design's liking with this set

Reply:
Note: we chatted out-of-band and decided to switch the selector to `[class*="narmi-icon-"]`